plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -284,7 +284,7 @@ public PostgreSqlClient(
             TypeManager typeManager,
             IdentifierMapping identifierMapping)
     {
-        super(config, "\"", connectionFactory, queryBuilder, identifierMapping);
+        super(config, "\"", connectionFactory, queryBuilder, identifierMapping, /*supportsFTE=*/true);
```

Comment:
Is the `/*supportsFTE=*/` for readability? We are not using such convention. FWIW you can configure IntelliJ to show hints regarding argument names for simple types.

Reply:
Yes it was for readability but i can remove it